src/components/Login/LoginModal.scss

```
@@ -0,0 +1,116 @@
+@import '/src/styles/variables.scss';
+@import 'https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.2/css/all.min.css';
+
+.loginWrapper {
+  position: fixed;
+  inset: 0;
+  @include flexCenter;
+  flex-direction: column;
+  background-color: rgba(0, 0, 0, 0.6);
+  transition: all 0.3s ease-in-out;
+  overflow: hidden;
+  z-index: 999;
```

```suggestion
.loginWrapper {
  @include flexCenter;
  position: fixed;
  inset: 0;
  flex-direction: column;
  background-color: rgba(0, 0, 0, 0.6);
  transition: all 0.3s ease-in-out;
  overflow: hidden;
  z-index: 999;
```
include는 스타일의 최상단으로 올려서 사용하는 것이 컨벤션입니다!

---

수정 했습니다